### PR TITLE
[WIP] Different Bleeding for Slippery Exotic Blood

### PR DESCRIPTION
**Pull request by Kiyahitayika**
## Please Note
Alternative to https://github.com/ParadiseSS13/Paradise/pull/14665 PR
This is not intended to "compete" with that PR and is not intended to be an attack or send any sort of "message". @Qwertytoforty and I were both involved in a conversation regarding Slime People blood and concerns over balance issues related to it a few days ago. I am in no way disagreeing with Qwerty's approach - in my honest opinion, I think it's a very pragmatic and fair way to go about addressing the concerns folks have raised.
 
It's become clear that, one way or another, several people are unhappy with the current status of Slime People's blood. However, it is also unclear how _many_  people are upset. Either way, the comments on Qwerty's PR and the discussion that ensued in #coding_chat on the Discord server following the posting of this WIP PR made it clear that this is **quite** a controversial issue.

The this PR addresses one of the issues raised (Slime Blood is Water and Water is slippery) with a different approach than https://github.com/ParadiseSS13/Paradise/pull/14665 (Qwerty's replaces the blood with Slime Jelly and mine makes it so a "fake" blood splatter is placed on the floor instead of any slippery reagent. Qwerty's PR also addresses other claims of balance issues related to Slime People blood.

This was originally meant for my own fork as a Proof of Concept to @SteelSlayer in discussion regarding Slime People blood - I I like to use projects to learn better ways of coding. I'm sure there are plenty of improvements to be made.

## What Does This PR Do
1.) Labels Reagents which make turf slippery (e.g. cause `MakeSlippery()`) with a new flag called `slippery`.
2.) Prevents `blood.dm` from affecting the turf when a species with an `exotic_blood` reagent that is `slippery` bleeds.
3.) Otherwise retains the turf reaction for `exotic_blood` species' blood (Slime People currently the only species using it)
4.) Creates new variable inside the `add_splatter_floor()` proc called `has_dna` which is used to prevent `exotic_blood` from putting DNA information into its blood splatters (i.e. forensic analyzers cannot detect dna from it, the "blood" cannot be used for Stable Mutagen or anything)
 TL;DR: Slime People retain water as blood, but bleeding produces blood splatter, not wet floor tiles.

To do: have blood color based on the body color (can do this on the species), make the fake blood decals for Slime People blood translucent (add alpha layer)

## Why It's Good For The Game
Okay, look, it makes Slime People not bleed Water, but they keep Water as blood. The fake blood they bleed doesn't contain DNA, so you can't yeet that blood into your syringe or whatnot and try to clone them with Stable Mutagen. Does it address other issues people see as unbalanced regarding Slime People blood being Water? No. 

## Images of changes
image

## Changelog
:cl:
add: New Reagent flag for "slippery" reagents
tweak: Changed how bleeding for exotic blood species. Because Slime People are the only species with exotic blood right now, the only effects are seen on their bleeding.
/:cl:

### CODE REVIEW

**Comment by Kiyahitayika** [comment]
This was supposed to be on my fork. Apologies.

**Comment by Kiyahitayika** [comment]
Closing this in order to throw full support behind https://github.com/ParadiseSS13/Paradise/pull/14665
